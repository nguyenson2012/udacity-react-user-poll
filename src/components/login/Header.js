import React from "react";
import { Typography, Divider } from "antd";

const Header = () => (
  <div style={{ textAlign: "center" }}>
    <Typography level={4}>Welcome to the User Poll Management App!</Typography>
    <Typography>Please sign in to continue</Typography>
    <Divider />
  </div>
);

export default Header;
