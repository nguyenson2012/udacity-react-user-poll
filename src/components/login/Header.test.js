import { render } from "@testing-library/react";
import Header from "./Header";

describe("Header", () => {
  it("will match snapshot", async () => {
    const component = render(<Header />);
    expect(component).toMatchSnapshot();
  });
});
