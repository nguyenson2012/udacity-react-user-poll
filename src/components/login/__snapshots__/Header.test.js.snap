// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header will match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="text-align: center;"
      >
        <article
          class="ant-typography css-dev-only-do-not-override-byeoj0"
          level="4"
        >
          Welcome to the User Poll Management App!
        </article>
        <article
          class="ant-typography css-dev-only-do-not-override-byeoj0"
        >
          Please sign in to continue
        </article>
        <div
          class="ant-divider css-dev-only-do-not-override-byeoj0 ant-divider-horizontal"
          role="separator"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="text-align: center;"
    >
      <article
        class="ant-typography css-dev-only-do-not-override-byeoj0"
        level="4"
      >
        Welcome to the User Poll Management App!
      </article>
      <article
        class="ant-typography css-dev-only-do-not-override-byeoj0"
      >
        Please sign in to continue
      </article>
      <div
        class="ant-divider css-dev-only-do-not-override-byeoj0 ant-divider-horizontal"
        role="separator"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
